deque.js

```diff
@@ -346,8 +346,17 @@ Deque.prototype.lastIndexOf = function (value, index) {
     return -1;
 }
 
-// TODO rename findValue
-Deque.prototype.find = function (value, equals, index) {
+Deque.prototype.find = function () {
+    if (
+        typeof console !== 'undefined' &&
+            typeof console.warn === 'function'
+    ) {
+        console.warn('.find function is deprecated please use findValue instead.');
+    }  
+    return Deque.prototype.findValue.apply(this, arguments);
```

Comment:
Bypassing dynamic dispatch is likely overkill here. If a derivative type overrides `find`, we should use it, but you’re right to ensure that you forward variadic args in case they extend the API. Consider…

```
this.find.apply(this, arguments);
```

Reply:
done in Array|Deque|SortedArray